recordset/recordset.html

```diff
@@ -2,101 +2,101 @@
-            <!-- pagination -->
-            <div style="margin-bottom: 10px;">
-                <span>{{ pageInfo.recordStart }} to {{ pageInfo.recordEnd }} of {{ vm.count }} Items</span>
-                <span class="dropdown" style="margin-left: 5px">
-                    <button class="btn btn-sm btn-default dropdown-toggle" type="button" data-toggle="dropdown">Items per page
-                        <span class="caret"></span></button>
-                    <ul class="dropdown-menu">
-                        <li><a href ng-click="pageLimit(10)">
-                            <span ng-show="pageInfo.pageLimit === 10" class="glyphicon glyphicon-ok"></span>
-                            <span ng-show="pageInfo.pageLimit !== 10" class="glyphicon glyphicon-invisible"></span>
-                            10</a>
-                        </li>
-                        <li><a href ng-click="pageLimit(25)">
-                            <span ng-show="pageInfo.pageLimit === 25" class="glyphicon glyphicon-ok"></span>
-                            <span ng-show="pageInfo.pageLimit !== 25" class="glyphicon glyphicon-invisible"></span>
-                            25</a>
-                        </li>
-                        <li><a href ng-click="pageLimit(50)">
-                            <span ng-show="pageInfo.pageLimit === 50" class="glyphicon glyphicon-ok"></span>
-                            <span ng-show="pageInfo.pageLimit !== 50" class="glyphicon glyphicon-invisible"></span>
-                            50</a>
-                        </li>
-                        <li><a href ng-click="pageLimit(75)">
-                            <span ng-show="pageInfo.pageLimit === 75" class="glyphicon glyphicon-ok"></span>
-                            <span ng-show="pageInfo.pageLimit !== 75" class="glyphicon glyphicon-invisible"></span>
-                            75</a>
-                        </li>
-                        <li><a href ng-click="pageLimit(100)">
-                            <span ng-show="pageInfo.pageLimit === 100" class="glyphicon glyphicon-ok"></span>
-                            <span ng-show="pageInfo.pageLimit !== 100" class="glyphicon glyphicon-invisible"></span>
-                            100</a>
-                        </li>
-                        <li><a href ng-click="pageLimit(200)">
-                            <span ng-show="pageInfo.pageLimit === 200" class="glyphicon glyphicon-ok"></span>
-                            <span ng-show="pageInfo.pageLimit !== 200" class="glyphicon glyphicon-invisible"></span>
-                            200</a>
-                        </li>
-                    </ul>
-                </span>
-            </div>
+        <!-- loading spinner -->
+        <div ng-show="pageInfo.loading" id="spinner">
+            <img src="spinner.gif" style="display: block; margin: 0 auto; width: 32px; height: 32px; opacity: 0.8;"/>
+            <div style="margin-top: 15px;">Loading...</div>
+        </div>
```

Comment:
@jessie can you create an issue for it?

On Aug 23, 2016 5:32 PM, "Jessie Wong" <notifications@github.com> wrote:

> In recordset/recordset.html
> <https://github.com/informatics-isi-edu/chaise/pull/546#discussion_r75977167>
> :
>
> > -                            <span ng-show="pageInfo.pageLimit !== 100" class="glyphicon glyphicon-invisible"></span>
> > -                            100</a>
> > -                        </li>
> > -                        <li><a href ng-click="pageLimit(200)">
> > -                            <span ng-show="pageInfo.pageLimit === 200" class="glyphicon glyphicon-ok"></span>
> > -                            <span ng-show="pageInfo.pageLimit !== 200" class="glyphicon glyphicon-invisible"></span>
> > -                            200</a>
> > -                        </li>
> > -                    </ul>
> > -                </span>
> > -            </div>
> > +        <!-- loading spinner -->
> > +        <div ng-show="pageInfo.loading" id="spinner">
> > +            <img src="spinner.gif" style="display: block; margin: 0 auto; width: 32px; height: 32px; opacity: 0.8;"/>
> > +            <div style="margin-top: 15px;">Loading...</div>
> > +        </div>
>
> @hongsudt <https://github.com/hongsudt> This div appears in almost all of
> our Chaise apps. We should refactor this styling out onto the main
> stylesheet, and perhaps this div into its own little spinner directive.
> Spinners aren't exactly high priority but maybe this could be a small issue.
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/informatics-isi-edu/chaise/pull/546/files/5454f69862efc1d6251ae913eafdf0169683170f#r75977167>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AAZUDHgnaOlmTw0cDzYi_tVFNtV0dHg-ks5qi5DGgaJpZM4JrMbs>
> .[image: Web Bug from
> https://github.com/notifications/beacon/AAZUDICxb5inFOeVslj3akZn7E6LbZtGks5qi5DGgaJpZM4JrMbs.gif]
> {"api_version":"1.0","publisher":{"api_key":"
> 05dde50f1d1a384dd78767c55493e4bb","name":"GitHub"},"entity":
> {"external_key":"github/informatics-isi-edu/chaise","
> title":"informatics-isi-edu/chaise","subtitle":"GitHub
> repository","main_image_url":"https://cloud.githubusercontent.com/assets/
> 143418/17495839/a5054eac-5d88-11e6-95fc-7290892c7bb5.png","
> avatar_image_url":"https://cloud.githubusercontent.com/
> assets/143418/15842166/7c72db34-2c0b-11e6-9aed-b52498112777.png
> ","action":{"name":"Open in GitHub","url":"https://github.
> com/informatics-isi-edu/chaise"}},"updates":{"snippets":[{"
> icon":"PERSON","message":"@howdyjessie in #546: @hongsudt This div
> appears in almost all of our Chaise apps. We should refactor this styling
> out onto the main stylesheet, and perhaps this div into its own little
> spinner directive. Spinners aren't exactly high priority but maybe this
> could be a small issue."}],"action":{"name":"View Pull Request","url":"
> https://github.com/informatics-isi-edu/chaise/pull/546/files/
> 5454f69862efc1d6251ae913eafdf0169683170f#r75977167"}}}